scattering/scattering.py

```diff
@@ -26,10 +26,12 @@ def structure_factor(trj, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False)
     Parameters
     ----------
     trj : mdtraj.Trajectory
-        A trajectory for which the structure factor is to be computed.
+        A trajectory for which the structure factor is to be computed.   
+    pair : array-like, shape=(2,), optional, default=('all', 'all')
```

Comment:
Above the default argument is specified as `None` but in the docstring you have the default as `('all', 'all')`